alpha/radio.cpp

```diff
@@ -22,6 +22,14 @@ _TS_RADIO::_TS_RADIO() {
   this->wifiTimerStart = -(WIFI_RETRY_INTERVAL);
 }
 
+bool _TS_RADIO::getUploadFlag()
+{
+	return uploadFlag;
+}
+const char * _TS_RADIO::getWiFiName() 
```

Comment:
nit: newline

Reply:
why do we need this fn ? WIFI_SSID is temporary

Reply:
Should uploadFlag be in EEPROM?

Reply:
Let me make the changes to TS_Storage to add the flag